### DISCUSSION TITLE
Can't start ssh session with -e

### DISCUSSION THREAD

**Discussion by caferen** [ghostty-org/ghostty#3691] [Feature Requests, Ideas]
I'd like to start an ssh session on a new window with `open -n /Applications/Ghostty.app --args -e ssh <remote>` so that the window does not parse the host shell config which spawns a tmux session. This allows me to start a tmux session on the remote device instead (host and remote share configs so same tmux bindings conflict).

However, windows spawned with this command will immediately exit as soon as the connection is established and the remote shell starts.

I've also tried (unsuccessfully):

* `open -n /Applications/Ghostty.app --args --quit-after-last-window-closed=false -e ssh <remote>`
* `open -n /Applications/Ghostty.app --args --initial-command="ssh <remote>"`
* `open -n /Applications/Ghostty.app --args --command="ssh <remote>"`

**Comment by pwfff** [comment]
had the same issue, which resolved when i installed the TERMINFO on the remote machine: https://ghostty.org/docs/help/terminfo#copy-ghostty's-terminfo-to-a-remote-machine

my command was `ghostty -e ssh me@server -t 'tmux attach -t sessionname -d'`

**Reply by taylrfnt** [reply]
Are you on Linux?  As far as I understand it, macOS users can't open ghostty via the CLI (yet - #2353), in the meantime, they have to use `open`.

**Comment by taylrfnt** [comment]
This is similar to #5575 , but I'm able to do this with one of two ways on my Mac (Sequoia 15.3):

**direct from the command line** 
 `open -n /Applications/Ghostty.app --args --command="ssh <remote>"`

**in ghostty configuration**
```ini
command = ssh <remote>
```

I'm running the following Ghostty version (slightly stale `tip` build) & my remote is a NixOS headless system w/ zsh:
```
❯ ghostty +version
Ghostty 1.1.3-main+da32534e

Version
  - version: 1.1.3-main+da32534e
  - channel: tip
Build Config
  - Zig version: 0.13.0
  - build mode : builtin.OptimizeMode.ReleaseFast
  - app runtime: apprt.Runtime.none
  - font engine: font.main.Backend.coretext
  - renderer   : renderer.Metal
  - libxev     : main.Backend.kqueue
  ```

**Reply by taylrfnt** [reply]
Oh, I missed the bit about tmux.  If I append `-t {options}` to the command, then yes, it does exit after the session is started.  I assume the command sends a valid exit code back to the host after tmux boots up
